src/session/session_api.c

```diff
@@ -2209,6 +2212,22 @@ __session_get_rollback_reason(WT_SESSION *wt_session)
     return (session->txn->rollback_reason);
 }
 
+/*
+ * __session_get_last_error --
+ *     WT_SESSION->get_last_error method.
+ */
+static void
+__session_get_last_error(WT_SESSION *wt_session, int *err, int *sub_level_err, const char **err_msg)
+{
+    WT_SESSION_IMPL *session;
```

Comment:
These lines can be combined to 

`    WT_SESSION_IMPL *session = (WT_SESSION_IMPL *)wt_session;`

Reply:
Done